### DISCUSSION TITLE
Gauge wit multiple needles (indicators)

### DISCUSSION THREAD

**Discussion by wanthalf** [HASwitchPlate/openHASP#609] [Documentation]
The documentation shows an image of a gauge with three needles (value indicators). However, there is no description on how to define and use multiple value indicators in OpenHASP.

**Comment by htvekov** [comment]
Hi' @wanthalf

Multiple needles aren't supported yet in openHASP.
Check Discord where this question has popped up a few times.
A solution could be to stack multiple needles.

https://discord.com/channels/538814618106331137/692393568266092617/1167141894153838695

**Reply by wanthalf** [reply]
Thanks. In that case it would be better to replace the confusing image in the documentation, at least. Even better to mention it explicitly.

**Reply by fvanroie** [reply]
Feel free to send a Pull request to the docs repository.

**Comment by RyanEwen** [comment]
Thought I was going crazy. This clears it up, I guess. Hope to see this feature come soon.

**Reply by wanthalf** [reply]
Here an example of a working configuration. The second "gauge" adds a second (blue) indicator on top of the primary gauge, the second "label" adds another label showing the second value below the central circle (not the most beautiful solution, I agree):

```
{"id":10,"obj":"obj","x":0,"y":30,"w":158,"h":158,"click":0,"bg_color":"#2C3E50","border_color":"#000000"}
{"id":11,"obj":"gauge","x":5,"y":7,"w":150,"h":150,"parentid":10,"min":-200,"max":400,"val":0,"format":1,"critical_value":301,"label_count":13,"line_count":61,"text_font":14,"border_width":0,"pad_top":2,"pad_bottom":2,"pad_left":2,"pad_right":2,"value_str":"°C","value_ofs_y":55,"value_font":16,"bg_opa":0,"line_width10":3,"line_rounded10":1,"line_color":"#348feb","line_color60":"#348feb","scale_grad_color":"#eb4934","scale_grad_color60":"#eb4934","scale_end_color":"#eb4934","scale_end_color60":"#eb4934"}
{"id":14,"obj":"gauge","x":5,"y":7,"w":150,"h":150,"parentid":10,"min":-200,"max":400,"val":0,"format":1,"critical_value":301,"label_count":13,"line_count":61,"text_font":14,"border_width":0,"pad_top":2,"pad_bottom":2,"pad_left":2,"pad_right":2,"value_str":"°C","value_ofs_y":55,"value_font":16,"bg_opa":0,"line_width10":3,"line_rounded10":1,"line_color10":"#00c0c0","line_opa60":0,"line_opa":0}
{"id":12,"obj":"obj","x":50,"y":50,"w":50,"h":50,"parentid":14,"click":0,"radius":30,"border_width":2,"border_opa":200}
{"id":13,"obj":"label","x":50,"y":62,"w":50,"h":28,"parentid":14,"text":"0.0","text_font":21,"align":"center"}
{"id":15,"obj":"label","x":50,"y":100,"w":50,"h":28,"parentid":14,"text":"0.0","text_font":19,"align":"center","text_color":"#00c0c0"}
```

**Reply by RyanEwen** [reply]
Thank you, I'll try this method
